docs/data-plane-configuration.md

```diff
@@ -0,0 +1,73 @@
+# Data Plane Configuration
+
+This document describes how to enable or customize various NGINX features.
+
+## Overview
+
+NGINX Gateway Fabric supports enabling or customizing various NGINX features via NginxProxy resource.
+The resource is created during the installation of NGINX Gateway Fabric. An update to the resource will make
+NGINX Gateway Fabric regenerate NGINX configuration and reload NGINX.
+
+If using manifests, the default name of the resource is `nginx-proxy-config`. If using Helm, the default name
+of the resource is `<release-name>-proxy-config`. It is deployed in the same Namespace as the controller
+(default `nginx-gateway`).
+
+The control plane only watches this single instance of the custom resource. If the resource is invalid per the OpenAPI
```

Comment:
Rather than "only watches this single instance of the custom resource", I'd mention that we only watch the one that's referenced in the associated GatewayClass for this control plane.